core/templates/dev/head/components/state/response_header_directive.html

```diff
@@ -80,9 +80,12 @@
     <em ng-if="getOutcome().newStateName">(<[getOutcome().newStateName]>)</em>
     <em ng-if="!getOutcome().newStateName">Nowhere yet...</em>
   </span>
-  <span ng-if="getOutcome() && (isOutcomeLooping() || (!isCorrect() && !getOutcome().dest && isInQuestionMode()))">
+  <span ng-if="getOutcome() && isOutcomeLooping() && !isInQuestionMode()">
```

Comment:
For consistency can we start this with getOutcome() && !isInQuestionMode(), and the next clause with getOutcome() && isInQuestionMode(), with the other checks coming after?

Reply:
Done.